examples/user_playlists_contents.py

```diff
@@ -25,7 +25,7 @@ def show_tracks(results):
             print(playlist['name'])
             print('  total tracks', playlist['tracks']['total'])
 
-            results = sp.playlist(playlist['id'], fields="tracks,next")
+            results = sp.playlist(playlist['id'], fields="tracks,next,items")
             tracks = results['tracks']
             show_tracks(tracks)
```

Comment:
Related to: https://github.com/spotipy-dev/spotipy/pull/1068#issuecomment-2089318419
```suggestion
            tracks = sp.playlist_tracks(playlist['id'], fields="items,next")
            show_tracks(tracks)
```